build.sbt

```diff
@@ -174,15 +174,11 @@ lazy val commonSettings =
         "-language:postfixOps",
         "-language:existentials",
         "-release",
-        "11"
+        "11",
```

Comment:
Shouldn't we use 17 here?

Reply:
We can do it. Now it's just a matter of changing it here to fully switch to Java 17. At the beginning I even made [that change](https://github.com/TouK/nussknacker/pull/8366/commits/adb2d9e7a408af6a0d4c746524cfbf70359f4a1f) but finally decided to stick with 11 to make it easier to upgrade Nussknacker in downstream projects.